### PR TITLE
Delete reminder

### PR DESCRIPTION
**Pull request by JoaoFSCruz**

### CODE REVIEW

File: resources/js/components/ShowReminderModal.vue [hunk 1]
```diff
@@ -42,12 +57,23 @@
     export default {
         data() {
             return {
-                reminder: null
+                reminder: null,
```
Comment (AlexOlival) [inline resources/js/components/ShowReminderModal.vue]: I wonder if it is safer to initialize this as an empty object `{}`.
Definitely something we should investigate.

**Review by AlexOlival — APPROVED** [review]
Looks great! :rocket: 
Road to MVP boys!

File: resources/js/components/ShowReminderModal.vue [hunk 2]
```diff
@@ -5,32 +5,47 @@
...
+                            <a class="text-sm font-semibold text-grey-dark underline cursor-pointer">EDIT</a>
+                        </div>
+                    </div>
+                </div>
+                <div v-else class=" pb-6 mt-2">
+                    <p class="text-4xl text-black font-bold mb-4">Are you sure you want delete the reminder?</p>
+                    <p class="text-xl text-grey">
+                        The reminder to be deleted is
+                        <span class="font-semibold">{{ reminder.title }}</span>.
+                    </p>
```
Comment (AlexOlival) [inline resources/js/components/ShowReminderModal.vue:43]: Is this necessary though?

File: resources/js/components/ShowReminderModal.vue (hunk 1 above)
Comment (JoaoFSCruz) [reply resources/js/components/ShowReminderModal.vue]: Definitely something we should investigate! I was in doubt whether to initialize with an empty object or not :dizzy_face:  In any case, we should look with other eyes :eyes:

Comment (JoaoFSCruz) [reply resources/js/components/ShowReminderModal.vue]: Definitely something we should investigate! I was in doubt whether to initialize with an empty object or not :dizzy_face: In any case, we should look with other eyes :eyes:

File: resources/js/components/ShowReminderModal.vue (hunk 2 above)
Comment (JoaoFSCruz) [reply resources/js/components/ShowReminderModal.vue:43]: I guess we'll have to arrange a UI/UX review at some point in the development where we'll tune it up.

**Pull request merged**